server/game/cards/08.6-SAT/KhalasarEnRazzia.js

```diff
@@ -0,0 +1,28 @@
+const _ = require('underscore');
+
+const DrawCard = require('../../drawcard.js');
+
+class KhalasarEnRazzia extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () =>
+                this.game.currentChallenge &&
+                this.game.currentChallenge.isAttacking(this) &&
+                this.bloodriderIsAttacking(),
+            match: card => card === this.controller.activePlot,
+            effect: ability.effects.modifyClaim(1)
+        });
+    }
+
+    bloodriderIsAttacking() {
+        return this.game.currentChallenge &&
+            _.any(this.game.currentChallenge.getParticipants(), card =>
```

Comment:
I believe @cryogen has officially announced underscore as deprecated, so this should be rewritten as

```javascript
return this.game.currentChallenge.getParticipants().some(card => ...);
```